lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
@chloealee you can just remove the changes made to this file and we can figure out this portion with the other issue (#6583)

`git checkout origin/dev -- lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc`